### ISSUE TITLE
FR: please co-publish on Open VSX Registry

### ISSUE THREAD

**Issue by GitMensch** [RechInformatica/rech-cobol-debugger#12]
Please make this extension available to [Open VSX](https://github.com/eclipse/openvsx) marketplace, this will also make it directly available for [VSCodium](https://vscodium.com/) and [Gitpod](https://www.gitpod.io/). Open VSX already has a bunch of [COBOL extensions](http://open-vsx.org/?search=cobol) including some debuggers, but this one is missing, and especially with its interesting configurable approach this would be very nice.
 
> Open VSX is a [vendor-neutral](https://projects.eclipse.org/projects/ecd.openvsx) open-source alternative to the [Visual Studio Marketplace](https://marketplace.visualstudio.com/vscode). It provides a server application that manages [VS Code extensions](https://code.visualstudio.com/api) in a database, a web application similar to the VS Code Marketplace, and a command-line tool for publishing extensions similar to [vsce](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#vsce).

There is a [guide for publishing](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions) and after the initial setup it is just a matter of `osvx publish` (similar to the MS marketplace). If you want to then there's also a GitHub action to auto-publish on tagging to both (or one) marketplace referenced there (but a manual upload is also possible).

**Comment by GitMensch** [comment]
ping, I guess @GustavoASC

**Comment by GustavoASC** [comment]
Hi @GitMensch,

Thanks for opening this issue!
We will publish it to Open VSX during this week. :+1:

**Comment by GustavoASC** [comment]
Hello,

Created https://github.com/open-vsx/publish-extensions/pull/344 in order to publish the extension to Open VSX.
For now, I did it manually, but let's look ahead to add an automatic publishing Github action.

According to [Open VSX Readme](https://github.com/open-vsx/publish-extensions), it will be live at 03:03 UTC after they accept the pull request.

Hope it helps! :smile:

**Comment by GitMensch** [comment]
As noted in their response it would be good to have it published directly, not from the publish-bot (so the process is the same as with the ms-marketplace).

**Comment by GustavoASC** [comment]
Yeah, I will use `npx ovsx publish` and after that I will close/cancel that PR, in case everything works fine.
I used this publish-bot for a personal extension once, so I thought it would be easier.

But `npx ovsx publish` really looks better.
I will comment here again as soon as I publish with `npx ovsx publish`

**Comment by GustavoASC** [comment]
The extension is now published and can be found here :smile:
https://open-vsx.org/extension/rechinformatica/rech-cobol-debugger

**Comment by GitMensch** [comment]
Marvelous!

**Comment by GitMensch** [comment]
You _may_ want to [claim its namespace](https://github.com/EclipseFdn/open-vsx.org/issues/new?labels=namespace&template=claim-namespace-ownership.md&title=Claiming+namespace+rechinformatica), more information can be found in the [Open-VSX Wiki](https://github.com/eclipse/openvsx/wiki/Namespace-Access#how-to-claim-a-namespace)

**Comment by GitMensch** [comment]
Please update - it is on 1.0.39 and therefore quite outdated.

You may want to use a [GitHub action](https://docs.github.com/en/actions) that allows publishing to Open VSX and the MS market place together by releasing/tagging; [HaaLeo/publish-vscode-extension](https://github.com/HaaLeo/publish-vscode-extension#readme) does this, for example.
